### PR TITLE
feat: use discord timestamp where applicable

### PR DESCRIPTION
**Pull request by favna**
Also addressed:
- Weirdly formatted PaginatedMessage template footers
- Poorly formatted best/worst comment of `reddituser`
- A typo `gmsOffset` -> `gmsOffset`

### CODE REVIEW

File: src/commands/Levelling/leaderboard.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class UserPaginatedMessageCommand extends PaginatedMessageCommand {
 			template: new MessageEmbed()
 				.setColor(await this.container.db.fetchColor(args.message))
 				.setTitle(args.t(LanguageKeys.Commands.Social.LeaderboardHeader, { guild: args.message.guild!.name }))
-				.setFooter(` - ${footerText}`, footerIcon)
+				.setFooter(`${footerText}`, footerIcon)
```
Comment (kyranet) [inline src/commands/Levelling/leaderboard.ts]:
```suggestion
				.setFooter(footerText, footerIcon)
```

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/commands/Misc/reddituser.ts [hunk 2]
```diff
@@ -81,31 +79,27 @@ export class UserPaginatedMessageCommand extends PaginatedMessageCommand {
 				embed
 					.addField(
 						`__${titles.bestComment}__`,
```
Comment (kyranet) [inline src/commands/Misc/reddituser.ts]: Why not use `underscore(titles.bestComment)`?

File: src/commands/Misc/reddituser.ts [hunk 3]
```diff
@@ -81,31 +79,27 @@ export class UserPaginatedMessageCommand extends PaginatedMessageCommand {
...
 	}
 
+	private formatCommentHeader(comment: Reddit.CommentDataElement): string {
+		return [
+			hyperlink('🔗', hideLinkEmbed(`https://reddit.com${comment.permalink}`)),
+			bold(comment.score.toString()),
+			inlineCode(`/r/${comment.subreddit}`),
+			'',
+			'-',
+			'',
```
Comment (kyranet) [inline src/commands/Misc/reddituser.ts]:
```suggestion
			'-',
```

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/commands/Tools/whois.ts [hunk 4]
```diff
@@ -61,8 +66,10 @@ export class UserCommand extends SkyraCommand {
 		const titles = t(LanguageKeys.Commands.Tools.WhoisMemberTitles);
 		const fields = t(LanguageKeys.Commands.Tools.WhoisMemberFields, {
 			member,
-			createdTimestampOffset: Date.now() - member.user.createdTimestamp,
-			joinedTimestampOffset: Date.now() - member.joinedTimestamp!
+			memberCreatedAt: time(secondsFromMilliseconds(member.user.createdTimestamp), TimestampStyles.ShortDateTime),
+			memberCreatedAtOffset: time(secondsFromMilliseconds(member.user.createdTimestamp), TimestampStyles.RelativeTime),
+			memberJoinedAt: time(secondsFromMilliseconds(member.joinedTimestamp!), TimestampStyles.ShortDateTime),
+			memberJoinedAtOffset: time(secondsFromMilliseconds(member.joinedTimestamp!), TimestampStyles.RelativeTime)
```
Comment (kyranet) [inline src/commands/Tools/whois.ts]: Please don't do this twice.

File: src/commands/Misc/reddituser.ts (hunk 2 above)
Comment (favna) [reply src/commands/Misc/reddituser.ts]: Forgot lol

File: src/listeners/guilds/channels/channelUpdateNotify.ts [hunk 5]
```diff
@@ -215,7 +215,10 @@ export class UserListener extends Listener<typeof Events.ChannelUpdate> {
 	}
 
 	private displayBitrate(t: TFunction, previous: number, next: number) {
-		return t(LanguageKeys.Events.Guilds.Logs.ChannelUpdateBitrate, { previous: previous / 1000, next: next / 1000 });
+		return t(LanguageKeys.Events.Guilds.Logs.ChannelUpdateBitrate, {
+			previous: secondsFromMilliseconds(previous),
+			next: secondsFromMilliseconds(next)
+		});
```
Comment (kyranet) [inline src/listeners/guilds/channels/channelUpdateNotify.ts]: Actually, this is bitrate (hertz), not milliseconds.

File: src/listeners/guilds/channels/channelCreateNotify.ts [hunk 6]
```diff
@@ -118,7 +118,7 @@ export class UserListener extends Listener<typeof Events.ChannelCreate> {
 	}
 
 	private displayBitrate(t: TFunction, value: number) {
-		return t(LanguageKeys.Events.Guilds.Logs.ChannelCreateBitrate, { value: value / 1000 });
+		return t(LanguageKeys.Events.Guilds.Logs.ChannelCreateBitrate, { value: secondsFromMilliseconds(value) });
```
Comment (kyranet) [inline src/listeners/guilds/channels/channelCreateNotify.ts]: Actually, this is bitrate (hertz), not milliseconds.

File: src/lib/util/common/times.ts [hunk 7]
```diff
@@ -9,6 +10,15 @@ export function seconds(seconds: number): number {
...
 }
 
+/**
+ * Converts a number of milliseconds to seconds.
+ * @param milliseconds The amount of milliseconds
+ * @returns The amount of seconds `milliseconds` equals to.
+ */
+export function secondsFromMilliseconds(milliseconds: number): number {
+	return roundNumber(milliseconds / Time.Second);
+}
```
Comment (kyranet) [inline src/lib/util/common/times.ts]: Missing unit tests

**Review by kyranet — CHANGES_REQUESTED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**